mysql-compatibility.md

```diff
@@ -45,7 +45,7 @@ However, some features of MySQL are not supported. This could be because there i
 
 ### Auto-increment ID
 
-+ In TiDB, auto-increment columns are only guaranteed to be unique and incremental on a single TiDB server, but they are *not* guaranteed to be incremental among multiple TiDB servers or allocated sequentially. It is recommended that you do not mix default values and custom values. Otherwise, you might encounter the `Duplicated Error` error message.
++ In TiDB, auto-incremental columns are globally unique, but only guaranteed to be incremental and sequential on a single TiDB server. They are *not* guaranteed to be incremental among multiple TiDB servers or allocated sequentially. It is recommended that you do not mix default values and custom values. Otherwise, you might encounter the `Duplicated Error` error message.
```

Comment:
How about this version? We also have one FAQ about this in https://docs.pingcap.com/tidb/stable/sql-faq#why-does-the-auto-increment-id-of-the-later-inserted-data-is-smaller-than-that-of-the-earlier-inserted-data-in-tidb ([Chinese version](https://docs.pingcap.com/zh/tidb/stable/sql-faq#tidb-%E4%B8%AD%E4%B8%BA%E4%BB%80%E4%B9%88%E5%87%BA%E7%8E%B0%E5%90%8E%E6%8F%92%E5%85%A5%E6%95%B0%E6%8D%AE%E7%9A%84%E8%87%AA%E5%A2%9E-id-%E5%8F%8D%E8%80%8C%E5%B0%8F))
```suggestion
+ In TiDB, auto-incremental columns are globally unique. They are incremental on a single TiDB server, but *not* necessarily incremental among multiple TiDB servers or allocated sequentially. It is recommended that you do not mix default values and custom values. Otherwise, you might encounter the `Duplicated Error` error message.
```